openhands/core/config.py

```diff
@@ -64,10 +64,10 @@ class LLMConfig:
     aws_access_key_id: str | None = None
     aws_secret_access_key: str | None = None
     aws_region_name: str | None = None
-    num_retries: int = 10
+    num_retries: int = 6
     retry_multiplier: float = 2
-    retry_min_wait: int = 3
-    retry_max_wait: int = 300
+    retry_min_wait: int = 4
+    retry_max_wait: int = 60
```

Comment:
I think we made them a bit higher to ensure that Anthropic LLMs on first tiers still work. They start with very low allowed requests. Can't really compare with next tiers, when they become more usable.

Reply:
The retry mechanic here is quite bad, imo, as there is ONLY "feedback" inside the terminal when an exception occurs, nothing in the UI until _after_ the last retry.
Waiting for the next retry for 5 minutes makes this unusable, no matter what LLM is being used.
I just tried this and even the reduced defaults are long for a user.

Reply:
I hear you, max 60 makes sense, indeed, there's a max _per minute_ that LLMs have. Not sure 300 ever helps...

Reply:
Even with the 10 retries, some of my evaluation process still can be broken due to the LLM rate limit (especially when running w/ multi-processing).

> as there is ONLY "feedback" inside the terminal when an exception occurs, nothing in the UI until after the last retry.

I think maybe we should *keep* these long retries and wait. - maybe the better way going forward is to figure out a way to display these errors into the FE UI (e.g., have a window pop-up potential warnings showing "your LLM API may get rate-limited : {execption}")? cc @amanape @rbren 

If we simply reduce the number of retries - it just forces some sessions to error out bc of the LLM rate limit, making the whole thing even more un-usable. WDYT?

Reply:
Then just set the values higher for evaluation runs, but as a _default_ the old values are bonkers.

Reply:
Additionally, what may help indirectly, will be to also allow the user to change these settings themselves in the UI. It won't help with the lack of info issue, but customizing them will be good. They really do differ a lot per provider and these pesky tiers.

Reply:
@xingyaoww - the user experience isn't helped with 20 minutes of nothing happening.
Evaluations are special scenarios and I'd put the burden to configure their retries accordingly, especially as they may differ vastly. Don't put these high wait values on the user. :)